pandas/tests/io/test_sql.py

```diff
@@ -64,6 +64,11 @@
 ]
```

Comment:
Can you remove the pytest marker on L63 that makes this run on a single CPU? That will be the true test if this is working or not